docs/source-pytorch/tuning/profiler.rst

```diff
@@ -4,6 +4,29 @@
 Find bottlenecks in your code
 #############################
 
+.. warning::
```

Comment:
lets move the note to this page
https://lightning.ai/docs/pytorch/stable/tuning/profiler_basic.html
that specifically has to do with the profiler feature in lightning and not have it in the overview page